docs/configure/ci/configuration.md

```diff
@@ -105,6 +105,15 @@ manifests:
       repo: https://helm.nginx.com/stable
 ```
 
+## Scanning Terraform Files
+You can scan Terraform files by specificing a list of directories to scan:
```

Comment:
update to "Scan Terraform files by specifying a list of directories to scan:"

Reply:
```suggestion
Scan Terraform files by specificing a list of directories to scan:
```

Reply:
@rbren update "specificing" to "specifying"